examples/notebooks/simulating-long-experiments.ipynb

```diff
@@ -0,0 +1,2011 @@
+    "n_Li_init = parameter_values.evaluate(param.n_Li_particles_init)\n",
+    "\n",
+    "esoh_sol = esoh_sim.solve(\n",
+    "    [0],\n",
+    "    inputs={\"V_min\": Vmin, \"V_max\": Vmax, \"C_n\": Cn, \"C_p\": Cp, \"n_Li\": n_Li_init},\n",
+    ")\n",
+    "print(f\"Initial negative electrode SOC: {esoh_sol['x_100'].data[0]:.3f}\")\n",
+    "print(f\"Initial positive electrode SOC: {esoh_sol['y_100'].data[0]:.3f}\")\n",
+    "\n",
+    "# Update parameter values with initial conditions\n",
+    "c_n_max = parameter_values.evaluate(param.c_n_max)\n",
+    "c_p_max = parameter_values.evaluate(param.c_p_max)\n",
+    "parameter_values.update(\n",
+    "    {\n",
+    "        \"Initial concentration in negative electrode [mol.m-3]\": esoh_sol[\"x_100\"].data[0] * c_n_max,\n",
+    "        \"Initial concentration in positive electrode [mol.m-3]\": esoh_sol[\"y_100\"].data[0] * c_p_max,\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "focused-substitute",
+   "metadata": {},
+   "source": [
+    "We can now simulate a single CCCV cycle using the `Experiment` class (see [this notebook](http://localhost:8888/notebooks/examples/notebooks/Getting%20Started/Tutorial%205%20-%20Run%20experiments.ipynb) for more details)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "religious-primary",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-05-21 19:02:48,269 - [NOTICE] simulation.solve(710): Cycle 1/1 (140.881 ms elapsed) --------------------\n",
+      "2021-05-21 19:02:48,270 - [NOTICE] simulation.solve(742): Cycle 1/1, step 1/4: Discharge at 1C until 3.0V\n",
+      "2021-05-21 19:02:48,396 - [NOTICE] simulation.solve(742): Cycle 1/1, step 2/4: Rest for 1 hour\n",
+      "2021-05-21 19:02:48,521 - [NOTICE] simulation.solve(742): Cycle 1/1, step 3/4: Charge at 1C until 4.2V\n",
+      "2021-05-21 19:02:48,693 - [NOTICE] simulation.solve(742): Cycle 1/1, step 4/4: Hold at 4.2V until C/50\n",
+      "2021-05-21 19:02:49,172 - [NOTICE] simulation.solve(837): Finish experiment simulation, took 1.043 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "pybamm.set_logging_level(\"NOTICE\")\n",
+    "\n",
+    "experiment = pybamm.Experiment([\n",
+    "    (f\"Discharge at 1C until {Vmin}V\",\n",
+    "     \"Rest for 1 hour\",\n",
+    "     f\"Charge at 1C until {Vmax}V\", \n",
+    "     f\"Hold at {Vmax}V until C/50\"\n",
+    "    )\n",
+    "])\n",
+    "sim = pybamm.Simulation(spm, experiment=experiment, parameter_values=parameter_values)\n",
+    "sol = sim.solve()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "heavy-crisis",
+   "metadata": {},
+   "source": [
+    "Alternatively, we can simulate many CCCV cycles. Here we simulate either 100 cycles or until the capacity is 80% of the initial capacity, whichever is first. The capacity is calculated by the eSOH model"
```

Comment:
think the next (commented out) cell can be deleted and the "References" cell after it